content/02.introduction.md

```diff
@@ -1,24 +1,27 @@
+They have been used in social network mining to classify nodes [@doi:10.1007/978-1-4419-8462-3_5] and create recommendation systems [@doi:10.1145/3184558.3186904].
 These graphs have also been used in natural language processing to interpret simple questions and use relational information to provide answers [@arxiv:1404.4326; @doi:10.1145/3038912.3052675].
-In a biomedical setting these graphs have been used to prioritize genes relevant to disease [@pmid:21121028; @doi:10.1101/651000; @doi:10.1109/CIBCB.2019.8791472; @doi:10.1093/bioinformatics/bty559], perform drug repurposing [@doi:10.7554/eLife.26726] and identify drug-target interactions [@doi:10.1371/journal.pcbi.1002574].
+In a biomedical setting, these graphs have been used to prioritize genes relevant to disease [@pmid:21121028; @doi:10.1101/651000; @doi:10.1109/CIBCB.2019.8791472; @doi:10.1093/bioinformatics/bty559], perform drug repurposing [@doi:10.7554/eLife.26726] and identify drug-target interactions [@doi:10.1371/journal.pcbi.1002574].
 
 Despite their utility, precisely defining a knowledge graph is a difficult task because there are multiple conflicting definitions [@raw:KG_DEF].
-For this review, we define a knowledge graph as the following: a resource that integrates single or multiple sources of information into the form of a graph.
-This graph allows for the capacity to make semantic interpretation, continuously incorporate new information and uncover novel hidden knowledge through computational techniques and algorithms.
-Based on this definition resources like Hetionet [@doi:10.7554/eLife.26726] would be considered a knowledge graph as Hetionet integrates multiple sources of information into the form of a graph (example shown in Figure {@fig:hetionet_schema}).
-Hetionet was used to derive novel information concerning unique drug treatments [@doi:10.7554/eLife.26726].
+For this review, we define a knowledge graph as the following: a resource that integrates single or multiple sources of information into the form a graph where nodes repesent entities and edges represent relationships that are shared between two entities.
+This definition is consistent with other definitions found in the literature [@doi:10.3233/SW-160218;10.1109/MIC.2019.2928449;@doi:10.1016/j.eswa.2019.112948;@doi:10.1109/MSN.2016.030;10.1109/jproc.2015.2483592;@doi:10.1145/1242572.1242667;@doi:10.1109/TKDE.2017.2754499].
+Often relationships are considered unidirectional (e.g., a compound treats a disease but a disease cannot treat a compound); however, there are cases where relationships can be considered bidirectional (e.g., a compound resembles another compound or a gene covaries with another gene).
+Based on our definition of a knowledge graph, graphs that we consider knowledge graphs may not be suitable for applications such as symbolic reasoning [@doi:10.1007/978-0-585-29599-2_11] but are still applicable for many biomedical applications.
```

Comment:
```suggestion
A subset of graphs that meet our definition of a knowledge graph would be unsuitable for methods such as symbolic reasoning [@doi:10.1007/978-0-585-29599-2_11], but we chose a more permissive definition because we identified numerous uses of these broadly defined graphs in the literature.
```

Reply:
See what you think of this. I'm not in love with it. 🤷‍♂️